meetings/2022-05-09.md

```diff
@@ -0,0 +1,61 @@
+# W3C Solid Community Group: Test Suite Panel
+
+* Date: 2022-05-09T14:00:00Z
+* Call: https://meet.jit.si/solid-test-suite
+* Chat: https://gitter.im/solid/test-suite
+* Repository: https://github.com/test-suite-panel
+
+
+## Present
+* [Sarven Capadisli](https://csarven.ca/#i)
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* name
+
+
+### Introductions
+* name: text
+
+---
+
+## Topics
+
+* SC: The Test Suite Panel is a "horizontal" panel. We need to revisit/clarify our understanding of the Test Suite Panel alongside other panels in the CG as well as contributors to specs and implementers.
+
+
+### Test Suite Panel Charter
+* SC: We need to stabilise a mutual understanding and a coordination plan. To advance our work, we need to agree on a charter that details its goals, scope, deliverables, decision policy (e.g., along the lines of https://github.com/solid/process/blob/main/notifications-panel-charter.md ).
+* SC: Here are some proposed actions. In these proposals, it is assumed that:
+  * Anyone can write a test.
+  * Anyone can contribute to the actions.
+  * Tests and reports that are not approved for publication at locations acknowledged by the Editors Team is out of scope for the Charter.
+  * Software maintainers give consent for reports to be published.
```

Comment:
in practice that would mean that ESS and CSS would ask to be removed from our results table with immediate effect, so then we might as well unpublish the whole results table, i.e. resign from our important mission

Reply:
So for this point as well, let's discuss this in the meeting, not take it as an assumption.

Reply:
It depends on PROPOSAL-1. For example, if publishing reports under solidproject.org is what's deemed to be authoritative (having met the criteria of PROPOSALs 1-3), then that's within the scope of the Charter. The Charter does not - and practically can't -  prohibit publication of reports anywhere on the Web, and so it is out of scope. Our goal is to coordinate so that reports can flow towards where they should be as far as the CG is concerned.

As I've mentioned elsewhere, it'd be appropriate - acceptable behaviour - for the CG to publish authoritative reports only with the consent of the software maintainer.

The assumptions are necessary for the PROPOSALs. I could've baked them into each PROPOSAL but there was no need repeat.

Yes, we can review all of this in the meeting.

Reply:
> in practice that would mean that ESS and CSS would ask to be removed from our results table with immediate effect,

@michielbdejong You have been asked repeatedly to not speak on behalf of other teams. You are spreading misinformation again, and this incident has been reported to the CoC team.

What the CSS team asks is a statement of quality and representativeness of the tests, as documented in:

- https://github.com/solid-contrib/test-suite/issues/138
- https://github.com/solid-contrib/test-suite/issues/139
- https://github.com/solid-contrib/test-suite/issues/140

Reply:
We did not have time to discuss this point in the meeting, but maybe we can resolve it in the same way, using Yvo's proposed way forward. Let's discuss in issues.